Alliance.Common/Utilities/PathHelper.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.IO;
+
+namespace Alliance.Common.Utilities
+{
+    public class PathHelper
+    {
+        /// <summary>
+        /// Creates a relative path from one file or folder to another.
+        /// </summary>
+        /// <param name="fromPath">Contains the directory that defines the start of the relative path.</param>
+        /// <param name="toPath">Contains the path that defines the endpoint of the relative path.</param>
+        /// <returns>The relative path from the start directory to the end path.</returns>
+        /// <exception cref="ArgumentNullException"><paramref name="fromPath"/> or <paramref name="toPath"/> is <c>null</c>.</exception>
+        /// <exception cref="UriFormatException"></exception>
+        /// <exception cref="InvalidOperationException"></exception>
+        public static string GetRelativePath(string fromPath, string toPath)
+        {
+            if (string.IsNullOrEmpty(fromPath) || string.IsNullOrEmpty(toPath))
+            {
+                throw new ArgumentNullException($"fromPath or toPath is null");
```

Comment:
String interpolation (`$`) is not useful here if you are not using variables

You can even do : 
```csharp
throw new ArgumentNullException(
            $@"Following parameters are null : 
            {(fromPath == null ? "fromPath" : "")}
            {(toPath == null ? "toPath":"")}"
        );
```